diff --git a/libs/common/src/billing/services/account/billing-account-profile-state.service.ts b/libs/common/src/billing/services/account/billing-account-profile-state.service.ts
index 654af40ffe..8c54eaaec2 100644
--- a/libs/common/src/billing/services/account/billing-account-profile-state.service.ts
+++ b/libs/common/src/billing/services/account/billing-account-profile-state.service.ts
@@ -57,8 +57,8 @@ export class DefaultBillingAccountProfileStateService implements BillingAccountP
     await this.stateProvider.getUser(userId, BILLING_ACCOUNT_PROFILE_KEY_DEFINITION).update(
       (_) => {
         return {
-          hasPremiumPersonally: hasPremiumPersonally,
-          hasPremiumFromAnyOrganization: hasPremiumFromAnyOrganization,
+          hasPremiumPersonally: true,
+          hasPremiumFromAnyOrganization: true,
         };
       },
       {
